src/script/media/MediaConstraintsHandler.test.ts

```diff
@@ -77,28 +77,77 @@ describe('MediaConstraintsHandler', () => {
   });
 
   describe('getScreenStreamConstraints', () => {
-    it('returns constraints to get the screen stream if browser supports getDisplayMedia', () => {
+    it('returns constraints to get the screen stream if browser supports getDisplayMedia in conference call', () => {
       const constraintsHandler = createConstraintsHandler();
 
-      const constraints = constraintsHandler.getScreenStreamConstraints(ScreensharingMethods.DISPLAY_MEDIA) as any;
+      const constraints = constraintsHandler.getScreenStreamConstraints(
+        ScreensharingMethods.DISPLAY_MEDIA,
+        true,
+      ) as any;
+
+      expect(constraints.audio).toBe(false);
+      expect(constraints.video.height).toEqual(
+        jasmine.objectContaining({ideal: jasmine.any(Number), max: jasmine.any(Number)}),
+      );
```

Comment:
Just a suggestion, any can be removed here and we can adjust the type:
```
      const constraints = constraintsHandler.getScreenStreamConstraints(
        ScreensharingMethods.DISPLAY_MEDIA,
        true,
      );

      expect(constraints?.audio).toBeFalsy();
      expect((constraints?.video as MediaTrackConstraints).height).toEqual(
        jasmine.objectContaining({ ideal: jasmine.any(Number), max: jasmine.any(Number) }),
      );
```
Please have a look at similar places also.

Reply:
fixed